### PR TITLE
fix: [iOS] On Course "Home" tab the row height

### PR DESCRIPTION
**Pull request by forgotvas**
[iOS] On Course "Home" tab the row height changes after a while https://github.com/openedx/openedx-app-ios/issues/483

### CODE REVIEW

File: Core/Core/Extensions/UIApplicationExtension.swift [hunk 1]
```diff
@@ -36,8 +45,7 @@ public extension UIApplication {
     }
     
     var windowInsets: UIEdgeInsets {
-        guard let windowScene = UIApplication.shared.connectedScenes.first as? UIWindowScene,
-              let window = windowScene.windows.first else {
+        guard let window = windows?.first else {
```
Comment (rnr) [inline Core/Core/Extensions/UIApplicationExtension.swift]: could we use here guard let window = `window` - property added on line 18?

**Review by rnr — APPROVED** [review]

File: Core/Core/Extensions/UIApplicationExtension.swift (hunk 1 above)
Comment (forgotvas) [reply Core/Core/Extensions/UIApplicationExtension.swift]: done

**Review by saeedbashir — APPROVED** [review]
It would be better if you can mention opededX PR or Issue it addresses in the PR description

**Comment by forgotvas** [comment]
> It would be better if you can mention opededX PR or Issue it addresses in the PR description

done

**Pull request merged**